lib/jellyfish_web/telemetry.ex

```diff
@@ -11,61 +12,73 @@ defmodule JellyfishWeb.Telemetry do
   @impl true
   def init(_arg) do
     children = [
-      # Telemetry poller will execute the given period measurements
-      # every 10_000ms. Learn more here: https://hexdocs.pm/telemetry_metrics
-      {:telemetry_poller, measurements: periodic_measurements(), period: 10_000}
-      # Add reporters as children of your supervision tree.
-      # {Telemetry.Metrics.ConsoleReporter, metrics: metrics()}
+      MetricsAggregator,
+      {TelemetryMetricsPrometheus, metrics: metrics(&last_value/2)}
     ]
 
     Supervisor.init(children, strategy: :one_for_one)
   end
 
-  def metrics do
+  # Phoenix by default uses the `summary` metric type in `LiveDashboard`,
+  # but `TelemetryMetricsPrometheus` doesn't support it, so we have to use `last_value` instead
+  def metrics(metric_type \\ &summary/2) do
```

Comment:
Could we please link Prometheus and TelemetryMetricsPrometheus guides on naming events, measurements etc.?